web/src/modules/feedback/components/FeedbackBoxContainer.js

```diff
@@ -60,189 +45,62 @@ type StateType = {|
-        feedbackCategory
-      })
-    } else if (type === EVENTS_ROUTE && path && title) {
-      return new FeedbackVariant({
-        label: t('contentOfEvent', { event: title }),
-        feedbackType: PAGE_FEEDBACK_TYPE,
-        feedbackCategory
-      })
-    } else if (type === POIS_ROUTE && path && title) {
-      return new FeedbackVariant({
-        label: t('contentOfPoi', { poi: title }),
-        feedbackType: PAGE_FEEDBACK_TYPE,
-        feedbackCategory
-      })
-    } else if ([WOHNEN_ROUTE, SPRUNGBRETT_ROUTE].includes(type) && alias && title) {
-      return new FeedbackVariant({
-        label: t('contentOfOffer', { offer: title }),
-        feedbackType: OFFER_FEEDBACK_TYPE,
-        feedbackCategory
-      })
-    } else if (type === SEARCH_ROUTE && query) {
-      return new FeedbackVariant({
-        label: `${t('searchFor')} '${query}'`,
-        feedbackType: SEARCH_FEEDBACK_TYPE,
-        feedbackCategory
-      })
-    } else if (type === DISCLAIMER_ROUTE) {
-      return new FeedbackVariant({
-        label: `${t('disclaimer')}`,
-        feedbackType: PAGE_FEEDBACK_TYPE,
-        feedbackCategory
-      })
-    } else {
-      return null
+  getFeedbackType = (): FeedbackType => {
+    const { path, alias, location } = this.props
+    const routeType = location.type
+
+    switch (routeType) {
+      case EVENTS_ROUTE:
+        return path ? PAGE_FEEDBACK_TYPE : EVENTS_FEEDBACK_TYPE
+      case OFFERS_ROUTE:
+        return alias ? OFFER_FEEDBACK_TYPE : OFFERS_FEEDBACK_TYPE
+      case DISCLAIMER_ROUTE:
+        return PAGE_FEEDBACK_TYPE
+      case POIS_ROUTE:
+        return path ? PAGE_FEEDBACK_TYPE : CATEGORIES_FEEDBACK_TYPE
+      case CATEGORIES_ROUTE:
+        return path ? PAGE_FEEDBACK_TYPE : CATEGORIES_FEEDBACK_TYPE
+      default:
+        return CATEGORIES_FEEDBACK_TYPE
     }
   }
 
   /**
    * Returns the data that should be posted to the feedback endpoint
    */
-  getFeedbackData = (selectedFeedbackOption: FeedbackVariant, comment: string): FeedbackParamsType => {
-    const { location, query, isPositiveRatingSelected, path, alias } = this.props
+  getFeedbackData = (comment: string): FeedbackParamsType => {
+    const { location, isPositiveRatingSelected, path, alias } = this.props
     const { city, language } = location.payload
-    const isOfferOptionSelected = selectedFeedbackOption.feedbackType === OFFER_FEEDBACK_TYPE
-    const feedbackAlias = alias || (isOfferOptionSelected && selectedFeedbackOption.alias) || ''
+    const feedbackType = this.getFeedbackType()
 
     return {
-      feedbackType: selectedFeedbackOption.feedbackType,
-      feedbackCategory: selectedFeedbackOption.feedbackCategory,
+      feedbackType: feedbackType,
       isPositiveRating: isPositiveRatingSelected,
       comment,
       permalink: path,
       city: city || INTEGREAT_INSTANCE,
       language: language || DEFAULT_FEEDBACK_LANGUAGE,
-      alias: feedbackAlias,
-      query
```

Comment:
Why did you remove this? Because we have no feedback for the search, right?

Reply:
`SearchFeedback.js` implements this property, but here it is not necessary.